e2e-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/ObservationDomainTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.providers.r4;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThrows;
+
+import java.util.UUID;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.api.MethodOutcome;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.client.interceptor.BasicAuthInterceptor;
+import ca.uhn.fhir.rest.server.exceptions.ResourceGoneException;
+import ca.uhn.fhir.rest.server.exceptions.ResourceNotFoundException;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.HumanName;
+import org.hl7.fhir.r4.model.IdType;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Quantity;
+import org.hl7.fhir.r4.model.Reference;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openmrs.module.fhir2.FhirConstants;
+
+public class ObservationDomainTest {
+	
+	private static final String SERVER_BASE = "http://localhost:8080/openmrs/ws/fhir2/R4";
+	
+	private static final FhirContext FHIR_CONTEXT = FhirContext.forR4();
+	
+	private static final IGenericClient CLIENT = FHIR_CONTEXT.newRestfulGenericClient(SERVER_BASE);
+	
+	private final String NON_EXISTING_OBSERVATION_UUID = UUID.randomUUID().toString();
+	
+	private static String observationUuid = "34d45fr6-f3a9-4f46-8341-15f0de3d8476";
+	
+	private static final String OBSERVATION_PATIENT_UUID = "58c57d25-8d39-41ab-8422-108a0c277d98";
```

Comment:
Does out docker instance come with any sample data.  If not, i think it will save us alot of time referencing existing patients than creating new ones each time we need a patient